app/views/shared/svg/_use_icon.html.erb

```diff
@@ -1,3 +1,3 @@
-<svg xmlns="http://www.w3.org/2000/svg" class="svg-icon svg-icon--<%= icon %><%= defined?(variant) ? " svg-icon--#{icon}--#{variant}" : '' %>">
+<svg xmlns="http://www.w3.org/2000/svg" class="svg-icon svg-icon--<%= icon %><%= defined?(variant) ? " svg-icon--#{icon}--#{variant}" : '' %><%= defined?(class_name) ? " #{class_name}" : '' %>">
```

Comment:
I was told by Tomas to use local_assigns.has_key?

http://stackoverflow.com/questions/2060561/optional-local-variables-in-rails-partial-templates-how-do-i-get-out-of-the-de